### PR TITLE
kernal dependency prompt issuse fixed

### PR DESCRIPTION
**Pull request by kishorereddygit**

### CODE REVIEW

**Review by Srinivasan-78 — COMMENTED** [review]
Do we have to use sudo systemctl reboot?
will it affect the installation process

File: dependencies.sh [hunk 1]
```diff
@@ -20,19 +17,16 @@ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o
   $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
 sudo apt-get update
 
-#docker installation
 sudo apt-get install docker-ce docker-ce-cli containerd.io -y
-sudo pip3 install docker-compose
+sudo pip3 install docker-compose
```
Comment (Srinivasan-78) [inline dependencies.sh:21]: spaces cause a lot of issues in bash scripts so, we'll have to be careful while misplacing them.

Comment (kishorereddygit) [reply dependencies.sh:21]: end of the line space can be ignored

**Comment by kishorereddygit** [comment]
> Do we have to use sudo systemctl reboot? will it affect the installation process
We have to sudo on reboot , reboot will effect installation

**Pull request merged**